content/en/blogs/cilium-as-cni.md

```diff
@@ -0,0 +1,230 @@
+---
+title: 'Cilium as the CNI: Deploy a Kubernetes Cluster with KubeSphere Installed'
```

Comment:
For a hands-on guide combining third-party toolkit(s) or other tutorials around Kubernetes concepts, it may be a good option to remove 'KubeSphere' from the title. Mentioning the brand frequently may sound pushy to developers; perhaps we can briefly introduce KubeSphere as a handy tool in the body and let the reader explore further if they like.  Just my personal thought. What do you think？

Reply:
I agree with you. Changes made in [this commit](https://github.com/kubesphere/website/pull/1824/commits/9a316ceb8b622c9b5c0fe15d988475289ad91639).